api/envoy/config/trace/v3/dynamic_ot.proto

```diff
@@ -29,9 +30,14 @@ message DynamicOtConfig {
 
   // Dynamic library implementing the `OpenTracing API
   // <https://github.com/opentracing/opentracing-cpp>`_.
-  string library = 1 [(validate.rules).string = {min_len: 1}];
+  string library = 1 [
+    deprecated = true,
```

Comment:
An observation is that we could set a message-level deprecated annotation, but I'm guessing we would have to update some of the code for warning on this use of deprecated. Not strictly needed here, as to use this filter you need to set some stuff. But, for an empty filter config it would be needed in the future.